torch/nested/_internal/nested_tensor.py

```diff
@@ -22,6 +29,21 @@ def get_tensor_symint(tensor, *, coeff=1):
     return tensor_symint
 
 
+# Utility for testing
```

Comment:
nit: if it's really just used for testing, imo it should be defined in a test file